### ISSUE TITLE
Creating a Calibration Module creates NullReferenceException

### ISSUE THREAD

**Issue by SorraTheOrc** [Avindr/MxM-IssueTracking#122]
When creating a new Calibration module the UI code throws a Null Reference. Workaround is to click off the calibration object then back again.

Repro:

1. Right click -> Create -> MxM -> Utility -> Calibration Module
2. Drag AnimData into slot
3. Answer yes to import callibration (though same problem when answering no)
4. No calibration UI and Console has error below
5. Click off the calibration module so Inspector displays something else, then come back and it works

```
NullReferenceException: Object reference not set to an instance of an object
MxMEditor.CalibrationModuleInspector.OnInspectorGUI () (at Assets/Plugins/MotionMatching/Code/MxM/Editor/Inspectors/CalibrationModuleInspector.cs:264)
UnityEditor.UIElements.InspectorElement+<>c__DisplayClass58_0.<CreateIMGUIInspectorFromEditor>b__0 () (at <208995a09fe148f5a6f6c571838c154f>:0)
UnityEngine.GUIUtility:ProcessEvent(Int32, IntPtr)
```

**Comment by Avindr** [comment]
Thanks, Can you just confirm which Unity version you are using?

**Comment by SorraTheOrc** [comment]
Unity Version: 2019.4.21f1
MxM Version: 2.2.9

**Comment by Avindr** [comment]
This is fixed for MxM v2.2.10 coming soon
